src/Designer/frontend/libs/studio-assistant/src/components/MessageColumn/Messages/AssistantMessage/AssistantMessage.module.css

```
@@ -0,0 +1,171 @@
+}
+
+.assistantContent pre code {
+  background-color: transparent;
+  padding: 0;
+  border: none;
+  font-size: inherit;
+  line-height: inherit;
+  color: #24292f;
+  display: block;
+  white-space: pre;
+  word-wrap: normal;
```

Stylelint (17.12.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)